server/test-remove-compose.yml

```diff
@@ -0,0 +1,88 @@
+name: recceiver-test-remove-compose
```

Comment:
Maybe you could use include (https://docs.docker.com/compose/how-tos/multiple-compose-files/include/) for parts of the compose setup